### ISSUE TITLE
Request: Single tap mod keys

### ISSUE THREAD

**Issue by gilescope** [TeXitoi/keyberon#62]
I am guessing I am not the only one to want to try this. Specifically I would like to try it on shift as that is the most commonly used modifier.

Maybe the easist way to do this is detect if a shift is pressed and relesed but no other key strokes have occured in the meantime. That would not require any timings. If that was detected we could wrap the next  keystroke down with a shift down before and a shift up immediately after. no need I would think to wait till the pressed key had arrisen.

Thoughts?

**Comment by TeXitoi** [comment]
Yup, a new "sticky" action, with something as you describe: if released before anything is released, release after the next key press.
